imageroot/bin/node_exporter_handler

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
Maybe we can also rename this script to something more generic like `metrics-exporter-handler`?
We are moving from node_exporter to netdata, but in the future ... who knows!